bin/interpreter/index.js

```diff
@@ -275,7 +275,7 @@ class Interpreter {
     obj.symbolTable.set(currentNode, value);
     if (caller) caller.symbolTable.set('hii', obj);
     if (obj.parent) obj.parent[obj.name] = obj;
-    return res.success(null);
+    return res.success(value);
```

Comment:
What does this change do?

Reply:
This returns the same value that gets assigned to the object property. Decided to add return values back to all nodes to allow assignment chaining, for example:
```
wacha a = wacha b = wacha c = 1
```

Reply:
Smart. Real Smart